tests/PhpSpreadsheetTests/Helper/HtmlTest.php

```diff
@@ -48,4 +48,23 @@ public function testLiTag(): void
 
         self::assertSame($expected, $actual->getPlainText());
     }
+
+    public function testSTag(): void
+    {
+        $html = new Html();
+        $input = 'Hello <s>test</s>world';
+        $richText = $html->toRichTextObject($input);
+        $elements = $richText->getRichTextElements();
+
+        self::assertSame(count($elements), 3);
+
+        self::assertSame($elements[0]->getText(), 'Hello ');
+        self::assertTrue($elements[0]->getFont() === null || !$elements[0]->getFont()->getStrikethrough());
```

Comment:
Currently, the returned font is indeed not null. However, the exact behavior doesn't seem to be documented. As the HTML snippet doesn't contain any font related information for this element, it may be possible/reasonable/expected that a (future) implementation just returns a null font, instead of some default initialised font. As long as it isn't documented that parsing html will always return a non-null font, I'm not sure it is a good idea to assert on it in a unit test.
Nevertheless, If you think it is a good idea to add the `assertNotNull`, I'm happy to change it that way.

Reply:
If we change the behavior in future, the updated test will fail. That is exactly what we want to happen when we change behavior, so that we can document the change as "breaking". So, please update the assertion as requested. If you think the documentation needs to be updated (I don't think it does), please add that to this PR.